x/group/types/config.go

```diff
@@ -0,0 +1,35 @@
+package types
```

Comment:
Let's not create a type package. Group follows the latest module directory guidelines: https://docs.cosmos.network/main/build/building-modules/structure